modules/evolution_techBidAdapter.md

```diff
@@ -0,0 +1,53 @@
+# Overview
+
+```
+Module Name: e_volution Bidder Adapter
+Module Type: Bidder Adapter
+```
+
+# Description
+
+Module that connects to e-volution-tech demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+                // Will return static test banner
+                {
```

Comment:
The response from your server when given these banner params does not have an `ad` property, so the `interpretResponse` function determines that it is invalid and no ad is shown.
```
[{"width":300,"height":250,"mediaType":"banner","requestId":"20437cf5b57b36","cpm":0.3,"ttl":120,"creativeId":"123","netRevenue":true,"currency":"USD","dealId":"HASH"}]
```

Also, all responses from your server need to be gzipped.

Reply:
+ adpropertie and gzip response